tailwind.config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
Por que tornou esse ignore ativo para todo o arquivo?

Reply:
Olá Luciano, tudo bem?
Foi uma correção proposta pela extensão de eslint que utilizo para o seguinte problema:

'module' is not defined.eslint[no-undef](https://eslint.org/docs/latest/rules/no-undef)

Este mesmo problema ocorre na parte inferior do arquivo. Nesses casos (quando o mesmo arquivo tem várias exceções desse tipo), para não repetir o mesmo código, opto por colocar essa exceção no topo.

Mas sendo um problema. Posso reverter :)

Reply:
Foi só para entender a motivação mesmo, por que estando dessa forma desabilita para todo o arquivo e antes era apenas para a linha.

Não tenho contexto se foi uma regra adicionada por causa de um problema ou se é padrão da stack React+Vite, então sempre questiono quando vejo alguma mudança. Por mim, não parece ser um problema.

Reply:
Está certíssimo!